src/main/java/com/greghaskins/spectrum/NotifyingBlock.java

```diff
@@ -13,6 +14,8 @@ static NotifyingBlock wrap(final Block block) {
     return (description, notifier) -> {
       try {
         block.run();
+      } catch (final AssumptionViolatedException assumptionViolation) {
+        notifier.fireTestAssumptionFailed(new Failure(description, assumptionViolation));
```

Comment:
Technically an assumption failure should be reported differently. The coverage tool made me notice this.